doc/gen-function-list.py

```diff
@@ -53,15 +55,50 @@ def add_nickname(gtype, a, b):
 
     type_map(type_from_name('VipsOperation'), add_nickname)
 
-    # add 'missing' synonyms by hand
-    all_nicknames.append('crop')
-
     # make list unique and sort
     all_nicknames = list(set(all_nicknames))
     all_nicknames.sort()
 
+    # make dict with overloads
+    overloads = {
```

Comment:
I don't think there's a way to automatically generate these overloads?

Reply:
(I copied and pasted them from the old function list, so maybe I forgot a few)

Reply:
Ooop, didn't post my comment. Sorry, no, I don't think so, unfortunately.